ext/openssl/openssl.c

```diff
@@ -6539,7 +7234,7 @@ PHP_OPENSSL_API zend_string* php_openssl_random_pseudo_bytes(zend_long buffer_le
 		|| ZEND_LONG_INT_OVFL(buffer_length)
 #endif
 			) {
-		zend_argument_error(NULL, 1, "must be greater than 0");
```

Comment:
Can you please retain this message? I believe this was pointed by Gabriel in an error message

Reply:
ok